### DISCUSSION TITLE
Make toolbar settings close on second click

### DISCUSSION THREAD

**Discussion by Marko64** [maxi-blocks/maxi-blocks#4691] [Ideas]
When you toggle any toolbar settings you can't close it by clicking the same button again. I don't know if it was always like this, but it's annoying. You need to click off the block to close it. 

Checked core Gutenberg blocks and Kadence - both close toolbar settings on the second click.

**Comment by kyrapieterse** [comment]
what do you mean? jam please 🙏

**Reply by Marko64** [reply]
Sure: https://jam.dev/c/8c18586a-9123-4a4a-82e0-2f6ba9f1c412

**Reply by kyrapieterse** [reply]
easy fix?

**Reply by Fybex** [reply]
> easy fix?

should be
